hydromt/gis/vector_utils.py

```diff
@@ -57,27 +58,22 @@ def nearest_merge(
     gpd.GeoDataFrame
         Merged GeoDataFrames
     """
+    # Get nearest index right
     idx_nn, dst = nearest(gdf1, gdf2)
     if not inplace:
         gdf1 = gdf1.copy()
     valid = dst < max_dist if max_dist is not None else np.ones_like(idx_nn, dtype=bool)
-    columns = gdf2.columns if columns is None else columns
     gdf1["distance_right"] = dst
     gdf1["index_right"] = -1
     gdf1.loc[valid, "index_right"] = idx_nn[valid]
```

Comment:
It's unclear to me if these should be in the output since they only seem to be used in the test, but since they were already there we'll just leave them for now.